lbrynet/daemon/Daemon.py

```diff
@@ -2427,19 +2405,26 @@ def jsonrpc_wallet_send(self, amount, address=None, claim_id=None):
             }
         """
 
+        amount = self.get_dewies_or_error("amount", amount)
         if address and claim_id:
             raise Exception("Given both an address and a claim id")
         elif not address and not claim_id:
             raise Exception("Not given an address or a claim id")
-        if amount < 0.0:
+
+        if not amount:
+            raise NullFundsError
+        elif amount < 0:
```

Comment:
Move the amount checks to come right after the amoutn variable is set.

Reply:
:+1: